articles/azure-maps/how-to-use-ts-rest-sdk.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Use Azure Maps TypeScript REST SDK
+titleSuffix: Microsoft Azure Maps
+description: Learn about the Azure Maps TypeScript REST SDK. See how to load and use this client library to access Azure Maps REST services in web or Node.js applications.
+author: dubiety
+ms.author: yuchungchen
+ms.date: 07/01/2023
+ms.topic: how-to
+ms.service: azure-maps
+services: azure-maps
+ms.custom: devx-track-js
+---
+
+# Use Azure Maps TypeScript REST SDK
+
+Azure Maps provides a collection of npm modules for the *Azure TypeScript REST SDK*. These modules consist of client libraries that make it easy to use the Azure Maps REST services in web or Node.js applications by using JavaScript or TypeScript. For a complete list of the available modules, see [JavaScript/TypeScript REST SDK Developers Guide](how-to-dev-guide-js-sdk.md).
+
+## Use the SDK in a web application
```

Comment:
Does "SDK" refer to the "Azure Maps JavaScript/TypeScript REST SDK"? In the article [JavaScript/TypeScript REST SDK Developers Guide (preview)](https://learn.microsoft.com/azure/azure-maps/how-to-dev-guide-js-sdk) we refer to this the first time as "The Azure Maps JavaScript/TypeScript REST SDK (JavaScript SDK)" then we subsequently refer to as **JavaScript SDK**. Do you think it makes sense to continue to do this? If not, how should we refer to it?

Reply:
Updated the subtitle to:
> Use the REST SDK in a web application

JavaScript SDK sounds a bit too generic and can be confused with the map control Web SDK.